src/lib.rs

```diff
@@ -1420,6 +1439,35 @@ impl LightningNode {
             .map_runtime_error_to(RuntimeErrorCode::OfferServiceUnavailable)
     }
 
+    /// List action required items.
+    ///
+    /// Returns a list of actionable items. They can be:
+    /// * Uncompleted offers (either available for collection or failed).
+    /// * Unresolved failed swaps.
+    /// * Available funds resulting from channel closes.
+    pub fn list_action_required_items(&self) -> Result<Vec<ActionRequiredItem>> {
+        let uncompleted_offers = self.query_uncompleted_offers()?;
+
+        let failed_swaps = self.get_unresolved_failed_swaps()?;
+
+        let available_channel_close_funds = self.get_node_info()?.onchain_balance;
```

Comment:
I'd call that on-chain balance rather than channel close funds, because that's what you are retrieving here. May prevent confusion in the future.

Reply:
At least currently, the only way for funds to get to that `onchain_balance` is through channel closes. I proposed this name because we can have other types of onchain funds (i.e., funds from expired swaps), which wouldn't be included here.

Reply:
I've renamed the variant to be what you propose @andrei-21. [811e841](https://github.com/getlipa/lipa-lightning-lib/pull/917/commits/811e84145e305f38d6f7a02e7e23e7efc534530c)